STYLE_GUIDE.md

```diff
@@ -15,11 +15,25 @@ void formatted_code_again;
 ```
 
 #### Python
-Python code should conform to [PEP8](https://www.python.org/dev/peps/pep-0008/).
 
 Addons uses [flake8](http://flake8.pycqa.org/en/latest/) to check pep8 compliance and 
 code analysis.
 
+Addon use [Black](https://black.readthedocs.io/en/stable/) to format our code.
```

Comment:
nit: Addons

Reply:
When reviewing pull requests and adding a nit, I think it would be nice to use the "suggest changes" feature of Github :) it makes faster iteration cycles.

Reply:
```suggestion
Addons use [Black](https://black.readthedocs.io/en/stable/) to format our code.
```